tests/Controller/DashboardStatsControllerTest.php

```diff
@@ -74,7 +74,7 @@ public function test_paused_status_is_reflected_if_all_master_supervisors_are_pa
         $masters = Mockery::mock(MasterSupervisorRepository::class);
         $masters->shouldReceive('all')->andReturn([
             (object) [
-                'status' => 'running',
+                'status' => 'paused',
```

Comment:
Why was this changed?

Reply:
Because now it only will show "paused" if all master supervisorrs are paused. Changed in behaviour because it's now unclear if you look at the dashboard. Now it shows "paused" while only one master supervisor is paused and everything is running. Therefore this pull request to show users (see images) if not all master supervisors are paused or not. Test was failing because he expected to be "paused" if only one is "paused" despite the wrong title that was indicating all master supervisors are paused.